docs/devel/api_changes.md

```diff
@@ -485,6 +485,17 @@ To regenerate them:
 hack/update-generated-deep-copies.sh
 ```
 
+Since our components still use internal apis, you may need to generate code
```

Comment:
#23768 is the main PR that is migrating to the new generator. Once this is merge, I will send a PR removing the old generator and we can get back to this PR - I would assume next week for that. Will keep you posted.